doc/radosgw/s3/bucketops.rst

```diff
@@ -288,6 +288,8 @@ Request Entities
 | ``Permission``            | String      | The permission given to the ``Grantee`` bucket.                                              |
 +---------------------------+-------------+----------------------------------------------------------------------------------------------+
 
+.. note:: For accounts users, the ``Owner`` and ``ID`` in ``AccessControlPolicy`` may be set to either the legacy user ID or the new account ID. Both are accepted for backward compatibility.
```

Comment:
I would suggest to link here to the accounts documentation. Can also do this myself in some future PR. Something like:
```console
.. note:: For :ref:`accounts users <radosgw-account>`, the [...]
```

Also, if touching this again, I suggest wrapping text lines before column 80.